### PR TITLE
feat(crons): Add topic to support monitor clock tasks

### PR DESCRIPTION
**Pull request by evanpurkhiser**
We will be using this topic for https://github.com/getsentry/sentry/issues/58410

### CODE REVIEW

File: schemas/monitors-clock-tasks.v1.schema.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          "description": "The check-in ID to mark as timedout.",
+          "type": "number"
+        }
+      },
+      "required": ["ts", "checkin_id"]
+    },
+    "MonitorsMarkMissing": {
+      "type": "object",
+      "title": "mark_missing",
+      "description": "Indicates a monitor ID that should be jk",
```
Comment (wedamija) [inline schemas/monitors-clock-tasks.v1.schema.json:37]: marked missed?

File: schemas/monitors-clock-tasks.v1.schema.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        "ts": {
+          "description": "The timestamp the clock ticked at.",
+          "type": "number"
+        },
+        "monitor_environment_id": {
+          "description": "The monitor environment ID to generate a missed check-in for.",
+          "type": "number"
+        }
+      },
+      "required": ["ts", "monitor_environment_id"]
```
Comment (wedamija) [inline schemas/monitors-clock-tasks.v1.schema.json:53]: Type should be required too I think?

File: schemas/monitors-clock-tasks.v1.schema.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        "ts": {
+          "description": "The timestamp the clock ticked at.",
+          "type": "number"
+        },
+        "checkin_id": {
+          "description": "The check-in ID to mark as timedout.",
+          "type": "number"
+        }
+      },
+      "required": ["ts", "checkin_id"]
```
Comment (wedamija) [inline schemas/monitors-clock-tasks.v1.schema.json:32]: Same here for type

File: schemas/monitors-clock-tasks.v1.schema.json (hunk 2 above)
Comment (evanpurkhiser) [reply schemas/monitors-clock-tasks.v1.schema.json:53]: yeah it should, thanks

**Review by wedamija — APPROVED** [review]

**Pull request merged**